optimize/backend/src/main/java/io/camunda/optimize/OptimizeTomcatConfig.java

```diff
@@ -77,7 +76,6 @@ public class OptimizeTomcatConfig {
             OptimizeResourceConstants.STATIC_RESOURCE_PATH,
             OptimizeResourceConstants.ACTUATOR_ENDPOINT,
             PanelNotificationConstants.SEND_NOTIFICATION_TO_ALL_ORG_USERS_ENDPOINT,
-            RestConstants.BACKUP_ENDPOINT,
```

Comment:
Should endpoints defined in actuator in the `dist` module be listed here or not?

Reply:
@grlimacan is removing this ok? should I add another rule for "backupHistory" (a new endpoint that was added) ?

Reply:
I am actually not famililar with the new TomcatConfig, @buccarel is probably the one that can answer this question best

Reply:
From what I'm understanding, the handler of such new endpoint is already implemented outside of the optimize module. This should mean that there's no need to declare the endpoint in optimize